templates/systemd

```diff
@@ -1,17 +1,19 @@
 [Unit]
 Description=etcd - highly-available key value store
 Documentation=https://github.com/coreos/etcd
+Documentation=man:etcd
 After=network.target
 Wants=network-online.target
 
 [Service]
 Environment=DAEMON_ARGS=
 Environment=ETCD_NAME=%H
-Environment=ETCD_DATA_DIR=/var/lib/etcd/
+Environment=ETCD_DATA_DIR=/var/lib/etcd/default
 EnvironmentFile=-/etc/default/%p
 Type=notify
 User=etcd
 PermissionsStartOnly=true
+#ExecStart=/bin/sh -c "GOMAXPROCS=$(nproc) /usr/bin/etcd $DAEMON_ARGS"
```

Comment:
if we're going to leave this in, we should probably make it tuneable with a default to nproc and uncomment it.

But that can be done in another PR.